app/guid-file/template.hbs

```diff
@@ -3,7 +3,7 @@
     @isMobile={{this.media.isMobile}}
 >
     <:header>
-        <h3 local-class='project-link-header'>
```

Comment:
Should this local-class have been removed? Its only job is to add some padding to the bottom, but still.

Reply:
This was removed by accepting the incoming changes from upstream but can add this back.

Reply:
This is re-added.